src/person-tracking/README.md

```diff
@@ -1,25 +1,43 @@
-# How to build
-```
-cd src/person-tracking
-npm install
-```
+# Intel® RealSense™ technology with JavaScript API for Linux OS
 
-# How to run samples
-```
-cd src/person-tracking
-npm run maketarball
-```
-The above step generates a person-tracking-xxx.tar.gz file.
+`node-person`: Node.js Person Detection/Person Tracking/Person Recognition/Persong Skeleton Tracking/Body Gesture Recognition/Face Tracking algorithm module based on Intel® RealSense™ technology.
```

Comment:
@halton Yes, they are said to be supported. And sorry for my misunderstood of taking Face Tracking and Person Recognition as the same feature, they are not actually. Then, that LGTM.